corehq/tabs/tabclasses.py

```diff
@@ -159,10 +159,27 @@ def _get_tools_items(self):
             'icon': 'icon-tasks fa fa-tasks',
             'show_in_dropdown': True,
         }]
-        if toggles.USER_CONFIGURABLE_REPORTS.enabled(self.couch_user.username):
-            # Only show for **users** with the flag. This flag is also available for domains
-            # but should not be granted by domain, as the feature is too advanced to turn
-            # on for all of a domain's users.
+        from corehq.apps.users.models import Permissions
+        # For SaaS envs, if we want to display the tab only if Feature Flag is
+        # enabled on domain and the role has the permission to edit_ucrs
+        show_on_sass_env = (
+            toggles.USER_CONFIGURABLE_REPORTS.enabled(self.domain)
```

Comment:
are we ever going to allow user-specific toggles of `USER_CONFIGURABLE_REPORTS` after this change? if not, I suggest that we explicitly restrict the namespace to domain here.